performance/benchmark.rb

```diff
@@ -5,13 +5,14 @@
 profiler = ThemeRunner.new
 
 Benchmark.ips do |x|
-  x.time = 60
+  x.time = 10
```

Comment:
I played with it a bit and it looks like 60s doesn't really generate more meaningful data than 10s, so +1 for this change IMO.